learn/core_concepts/documents.md

```diff
@@ -169,6 +169,18 @@ There are several ways to set the primary key for an index:
 - You can set it manually [on document addition](/reference/api/documents.md#add-or-replace-documents)
```

Comment:
Would be nice to update the link in this line and the one above to point to the new sections, instead of the API reference.

Reply:
Thanks for pointing this out! I was supposed to do this